### DISCUSSION TITLE
Where is "Models" in widget settins in RC4?

### DISCUSSION THREAD

**Discussion by auceto** [EdgeTX/edgetx#2692] [Q&A] [answered]
Hi at all,
I've update my TX16s from 2.7 to 2.8 RC4.
In the radio screen the model images of my choice appeared correctrly.
If I use Screen Settings and I attempt to change the model image in the widget (before I've changed the image in the model settings of course) then "Models" command in the widget menù do not appear as in previus versione 2.7.
Why?
Is this a bug or I'm doing a mistake?

thanks

**Comment by pfeerick** [comment, marked as answer]
Ah, sorry, it's called "Model info" now...

**Reply by auceto** [reply]
Thank for reply.
Why "Model info"? It would be better "Model image".
  I used it with no seccess using my custom image.
But if I assign the same image using "Radio setting", select SD and selecting the image from Images folder the image it is assign correctly. Buuut you can't see the image immediately in "Select model", you have to do a radio reboot to see the image here.
With old 2.7 version assign image from widget was ok

**Reply by pfeerick** [reply]
Because it shows the model name and/or image. 

So with RC4, if you have the widget loaded, and assign an image via the Model settings -> Model image option, the image isn't  showing for you until you reboot the radio? I can't reproduce this atm on mine... as soon as I set the image either there, or assign it via the radio settings -> sd browser it shows up on the widget.

**Reply by auceto** [reply]
So if I have the widget loaded the assigned image do not appear and do not appear the model name too if as you said above the new command should be shows model name and/or Image.
to "force" the image I have to go  in Radio Settings e choice the image from SD card Image folder using "Assign bitmat" command; in this way I see the image everywhere.
but if assign the image in the widget using Screen setting:
1) the image do not appear in the widget
2) the image not appear in the model list when I use "Select model"; if I reboot the radio the image do not appear in the model list.
now I assign the image from SD card directly and
1) I see the image in the loaded widget
2) I do not see the image in the model list
3) If I reboot the image re appear everywhere

the image that I use is a custom image not default image.
I prove it again and this happen only with my custom image.
I don't know how to post this image here to allow you to take the test
